app/Resources/views/line/bundle_card.html.twig

```diff
@@ -1,47 +1,62 @@
-<div class="bundle-card-line{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
-    <div class="bundle-card-line-data">
-        <div class="col-sm-8 col-md-9 col-lg-10 bundle-card-line-content">
-            <h2>
-                <a href="https://packagist.org/packages/{{ ez_field_value(content, 'bundle_id') }}" title="See on Packagist.org">{{ ez_render_field(content, 'bundle_id') }}</a>
-            </h2>
-            <div class="bundle-card-line-description">
-                {{ ez_render_field(content, 'description') }}
+<div class="bundle-card-line-container{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
+    <div class="bundle-card-line">
+        <div class="bundle-card-line-data">
+            <div class="bundle-card-line-information">
+                <h1>
+                    {{ ez_render_field(content, 'packagist_url') }}
+                </h1>
+                <div class="bundle-card-line-description">
+                    {{ ez_render_field(content, 'description') }}
+                </div>
             </div>
-        </div>
-        <div class="col-sm-4 col-md-3 col-lg-2 bundle-card-line-author">
-            <div class="bundle-card-line-author-container">
-                {% if packageDetails.author[0] is defined %}
-                    <div class="bundle-card-line-author-username">{{ packageDetails.author[0].name }}</div>
-                {% endif %}
-                {% if packageDetails.authorAvatarUrl is defined and packageDetails.author[0].name is defined %}
-                    <div class="bundle-card-line-author-photo">
-                        <img src="{{ packageDetails.authorAvatarUrl }}?s=100" alt="{{ packageDetails.author[0].name }}" />
-                    </div>
-                {% endif %}
-                {% if packageDetails.updated is defined %}
-                    <div class="bundle-card-line-last-update">Updated: <span>{{ packageDetails.updated|date("d / m / Y") }}</span></div>
-                {% endif %}
+            <div class="bundle-card-line-counters">
+                <ul>
+                    <li class="downloads"><i class="fa fa-cloud-download" aria-hidden="true"></i> {{ ez_field_value(content, 'downloads') }}</li>
+                    <li class="stars"><i class="fa fa-star" aria-hidden="true"></i> {{ ez_field_value(content, 'stars') }}</li>
```

Comment:
In the jira issue (https://jira.ez.no/browse/COM-20010), I associated `stars` to `popularity`. Does it make sense?
@damianz5 @kmadejski What do you think?